src/ray/gcs/redis_context.cc

```diff
@@ -306,14 +310,20 @@ Status AuthenticateRedis(redisAsyncContext *context, const std::string &password
 
 void RedisAsyncContextDisconnectCallback(const redisAsyncContext *context, int status) {
   RAY_LOG(DEBUG) << "Redis async context disconnected. Status: " << status;
-  // Reset raw 'redisAsyncContext' to nullptr because hiredis will release this context.
-  reinterpret_cast<RedisAsyncContext *>(context->data)->ResetRawRedisAsyncContext();
```

Comment:
can you remove method RedisAsyncContext::ResetRawRedisAsyncContext since now nobody calls it ?